BrainPortal/db/migrate/20191025141308_add_position_to_users.rb

```diff
@@ -0,0 +1,20 @@
+class AddPositionToUsers < ActiveRecord::Migration[5.0]
+
+  def up
+    add_column :users, :position, :string, :after => :full_name
+
+    User.all.each do |user|
+      user.position = extract_position_from_log(user.getlog || "") || "Unknown"
+      user.save
+    end
+  end
+
+  def extract_position_from_log(log)
+    log[/Position:\s*(\.+?)\s*\z/, 1]
```

Comment:
OK I found the error: I made the mistake when I created this regex. It should be:

```ruby
  log[/Position:\s*(.+?)\s*$/m, 1]
```

There are three differences: no backslash before the period (duh), use `$` instead of `\z` (double duh) and use the multiline modifier `m` (optional but sensical).